pkg/virt-controller/watch/vm.go

```diff
@@ -1419,16 +1419,18 @@ func (c *VMController) stopVMI(vm *virtv1.VirtualMachine, vmi *virtv1.VirtualMac
 	}
 
 	// stop it
+	c.expectations.ExpectDeletions(vmKey, []string{controller.VirtualMachineInstanceKey(vmi)})
```

Comment:
If you move the expectation you should delete the expectation in case of patch error, similar to line 1439

Reply:
Ah I see, in that case I will move it back to how it was before.